### ISSUE TITLE
Fake Bot

### ISSUE THREAD

**Issue by User123456789012345678901234567890** [TemuLinkHack/TemuHack#2]
you all are gonna get tricked since this is basically a referral link so the person who created this page can earn more money, there is no software....(theres nothing located in the source code.)

**Comment by ghost** [comment]
Yeah, it's obviously bullshit. I don't know how people still fall for things like this.
